lalrpop-test/src/cfg.lalrpop

```diff
@@ -12,3 +12,33 @@ pub Created: () = {
     "x" => ()
 };
 
+#[cfg(not(feature = "test-not-set"))]
+pub CreatedWithNot: () = {
+    "x" => ()
+};
+
+#[cfg(not(feature = "test-set"))]
+pub NotCreatedWithNot: () = {
+    "x" => ()
+};
+
+#[cfg(any(feature = "test-not-set", feature = "test-set"))]
+pub CreatedWithAny: () = {
+    "x" => ()
+};
+
+#[cfg(any(feature = "test-not-set"))]
+pub NotCreatedWithAny: () = {
+    "x" => ()
+};
+
+#[cfg(all(feature = "test-set"))]
+pub CreatedWithAll: () = {
+    "x" => ()
+};
+
+#[cfg(all(feature = "test-set", feature = "test-not-set"))]
+pub NotCreatedWithAll: () = {
+    "x" => ()
+};
+
```

Comment:
It would be nice to also have some tests around the intersection of `cfg` with macro expansion.  Eg, using attributes inside a macro definition, or a conditionally defined macro.

Also for precedence interaction with `cfg`.

I'm just looking at the ordering change in normalization and wanting to make sure we have test coverage for moving `cfg` processing earlier.  Macros and precedence are the things that jump out to me as possible problem sources (although I can't think of anything specific I'd expect to go wrong there).

Reply:
good catch, also should test how it interferes with inline possibly.